barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover_mega.cpp

```diff
@@ -5,5 +5,5 @@
 #include "protogalaxy_prover_impl.hpp"
 namespace bb {
 
-template class ProtoGalaxyProver_<ProverInstances_<MegaFlavor, 2>>;
+template class ProtogalaxyProver_<ProverInstances_<MegaFlavor, 2>>;
```

Comment:
I didnt actually ask, maybe you know, why did we move all the PG logic in header files?

Reply:
I think Adam did this when he was cleaning up how we did instantiation.